src/RaisedButton/RaisedButton.js

```diff
@@ -78,7 +78,7 @@ function getStyles(props, context, state) {
     label: {
       position: 'relative',
       opacity: 1,
-      fontSize: '14px',
+      fontSize: raisedButton.fontSize || button.fontSize,
```

Comment:
Now that I look at it, there's no point in keeping `|| button.fontSize`, since `raisedButton.fontSize` will always be defined, right? Unless someone purposefully sets it to something falsy.